packages/shopify-codemod/test/fixtures/conditional-assign-to-if-statement/assign-to-other-identifier.input.js

```diff
@@ -0,0 +1,7 @@
+this.foo || (this.bar = 'lol');
```

Comment:
It spits out this:
```
if (!(this.foo || this.bar)) {
  this.baz = 'qux';
}
```

That's not pretty, but when I transform the decaffed admin/sello code, I don't see any instances of this.  My preference would be to not worry about it until we see it somewhere.